docs/csharp/quick-starts/numbers-in-csharp-local.md

```diff
@@ -0,0 +1,373 @@
+c = a / b;
+Console.WriteLine(c);
+```
+
+Run this code by typing `dotnet run` in your command window. 
+    
+You can also experiment by performing multiple mathematics operations in the same line, if you'd like.
+
+> [!TIP]
+> As you explore C# (or any programming language), you'll
+> make mistakes when you write code. The **compiler** will
+> find those errors and report them to you. When the output
+> contains error messages, look closely at the example code,
+> and the code in the interactive window to see what to fix.
+> That exercise will help you learn the structure of C# code.     
+
+You've finished the first step. Before you start the next section, let's move the current code into a separate method. That makes it easier to start working with a new example. Rename your `Main` method to `WorkWithIntegers` and write a new `Main` method that calls `WorkWithIntegers`. When you have finished, your code should look like this:
+
+```csharp
+using System;
+
+namespace NumbersInCSharp
+{
+    class Program
+    {
+        static void WorkingWithIntegers()
+        {
+            int a = 18;
+            int b = 6;
+            int c = a + b;
+            Console.WriteLine(c);
+            c = a - b;
+            Console.WriteLine(c);
+            c = a * b;
+            Console.WriteLine(c);
+            c = a / b;
+            Console.WriteLine(c);
+        }
+
+        static void Main(string[] args)
+        {
+            WorkingWithIntegers();
+        }
+    }
+}
+```
+
+## Explore order of operations
+
+Comment out the call to `WorkingWithIntegers()`. It will make the output less cluttered as you work in this section:
+
+```csharp
+//WorkingWithIntegers();
+```
+
+The `//` starts a **comment** in C#. Comments are any text you want to keep in your source code, but not execute as code. The compiler does not generate any executable code from comments.
+
+The C# language defines the precedence of different mathematics operations
+with rules consistent with the rules you learned in mathematics.
+Multiplication and division take precedence over addition and subtraction.
+Explore that by adding the following code to your `Main` method, and execuing
+`dotnet run`:
+
+```csharp
+int a = 5;
+int b = 4;
+int c = 2;
+int d = a + b * c;
+Console.WriteLine(d);
+ ```
+
+The output demonstrates that the multiplication is performed before the addition.
+
+You can force a different order of operation by adding parentheses around
+the operation or operations you want performed first. Add the following
+lines and run again:
+
+```csharp
+d = (a  + b) * c;
+Console.WriteLine(d);
+```
+
+Explore more by combining many different operations. Add something like
+the following lines at the bottom of your `Main` method. Try `dotnet run` again.
+
+```csharp
+d = (a + b) - 6 * c + (12 * 4) / 3 + 12;
+Console.WriteLine(d);
+```
+
+You may have noticed an interesting behavior for integers. Integer
+division always produces an integer result, even when the mathematical
```

Comment:
even when the mathematical result would... --> even when you'd expect the result to include a decimal or fractional portion.

Reply:
Good comment. I also changed the YML version that had the same text.